apps/web/public/static/locales/en/common.json

```diff
@@ -2439,5 +2439,8 @@
   "csat_score": "CSAT Score",
   "lockedSMS": "Locked SMS",
   "signing_up_terms": "By proceeding, you agree to our <1>Terms</1> and <2>Privacy Policy</2>.",
+  "unable_to_subscribe_to_the_platform": "Unable to subscribe to the platform, please try again later",
```

Comment:
@anikdhabal what do you think about `Unable to subscribe to the platform plan, please try again later` here?